client/src/components/App.jsx

```diff
@@ -97,4 +97,63 @@ const App = class extends React.Component {
   }
 };
 
+App.propTypes = {
+  aggregateFiveStarReview: PropTypes.number,
```

Comment:
would it be worthwhile to abstract all the aggregate star ratings into its own object?